### ISSUE TITLE
[Bug] Graphql: Unexpected token m in JSON at position 0

### ISSUE THREAD

**Issue by vdwpsmt** [yesoreyeram/grafana-infinity-datasource#296]
**Describe the bug**
when posting a graphql, response payload is not json but plain text.  This causes "Unexpected token" error. 

**Screenshots**
![image](https://user-images.githubusercontent.com/41926598/161800369-29344d77-af65-41de-9742-eb9b6be8336f.png)

**Version Details:**

- Grafana version : 8.4.3
- Plugin version : 0.8.3

**Additional context**
![image](https://user-images.githubusercontent.com/41926598/161800924-9d2b56f4-a027-4830-ae3a-fbceb770c0d9.png)
Response header in the infinity graphql data source plugin has the key value `Content-Type: text/plain; charset=utf-8`.
when executing the same query from another graphql client, response is json.  
Difference with infinity: in the request headers I can see a key `accept: application/json, text/plain, */*` 
I tried overwriting this in the interface but it doesn't seem to accept this.

**Comment by yesoreyeram** [comment]
I doubt the response headers of the api. Usually infinity respects response headers. Can you check the api response headers?

**Comment by vdwpsmt** [comment]
Indeed, probably nothing wrong there.  I found something else however.  I tested with a public available  graphql server
https://countries.trevorblades.com/ on the grafana demo server of infinity (https://grafana-infinity-datasource.herokuapp.com/) and compared with my own grafana server.  Result: 
- infinity server: works fine, as expected (json response returned)
- own server: same error caused by plain text returned instead of json.

then I updated my own server to the same grafana version (latest: 8.4.5) and now everything seems to work fine.
